### PR TITLE
Githubaction/to run pytest

### PR DESCRIPTION
**Pull request by Thoritie**
Purpose of this pull request
- To Execute Django tests in GitHub Actions #4 
https://github.com/dataength/dataengnews/issues/4

### CODE REVIEW

**Comment by Thoritie** [comment]
@zkan Please review 🙏🏻

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install flake8 pytest
```
Comment (zkan) [inline .github/workflows/tests.yml:36]: I guess we don't need `flake8` here.

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+      - name: Test with pytest
```
Comment (zkan) [inline .github/workflows/tests.yml:38]:
```suggestion
      - name: Test with Pytest
```

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+name: Run test
```
Comment (zkan) [inline .github/workflows/tests.yml:9]:
```suggestion
name: Run Tests
```

**Review by zkan — COMMENTED** [review]
Thanks. I'd request for some changes.

**Comment by Thoritie** [comment]
@zkan  Thank you 🎉, i just updated following you advice

**Review by zkan — APPROVED** [review]

**Pull request merged**